tidb-cloud/tidb-cloud-tls-connect-to-dedicated-tier.md

```diff
@@ -0,0 +1,321 @@
+---
+title: TLS connection to Dedicated Tier
+summary: Introduce TLS connection in TiDB Dedicated Tier.
+---
+
+# TLS connection to Dedicated Tier
+
+On TiDB Cloud, TLS connections are one of the basic security practices for connecting to Dedicated Tier clusters. We support you to configure multiple TLS connections from client, application and development tools to protect your data transmission security.
+
+Each Dedicated Tier cluster server TLS certificate is hosted on [AWS Certificate Manager (ACM)](https://aws.amazon.com/certificate-manager/), ACM Private CA keys are stored securely in AWS managed hardware security modules (HSMs) that adhere to [FIPS 140-2 Level 3](https://csrc.nist.gov/projects/cryptographic-module-validation-program/Certificate/3139) security standards in order to protect your TLS private keys when Dedicated Tier request AWS ACM to create it.
+
+> **Note:**
+>
+> Best practices for network access security include three steps:
+> Step1: Set up [IP Access List](/tidb-cloud/configure-ip-access-list.md)
+> Step2: Set up [Private Link](/tidb-cloud/set-up-private-endpoint-connections.md)
+> Step3: Set up TLS Connection
```

Comment:
Question is when using private link, still necessary to use TLS? And if it is, have we tested and provide steps on how to set TLS with private link? Not only AWS, but also GCP.

Reply:
sound good，how to set private link refer private link, already linked, Currently ,we don't support GCP Private link

Reply:
But the link does not show how to set TLS under private link. :-(
I know, we don't support private link on GCP now. So it this still our recommendation?